clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/provider/agent/Namespace.groovy

```diff
@@ -38,7 +38,7 @@ enum Namespace {
   RESERVATION_REPORTS,
   RESERVED_INSTANCES
 
-  final String ns
+  public final String ns
```

Comment:
In Groovy, members are public-by-default, so this isn't necessary.

Reply:
But when you access it from java, it needs the specific declaration of `public` :( w/o explicitly declaring it public it won't compile